scripts/tutorials/generate_database.py

```diff
@@ -7,14 +7,34 @@
 sys.path.append(os.getcwd())  # hack to be able to discover simple
 from simple.schema import *
 
-DB_NAME = 'SIMPLE.db'
+# Location of source data
 DB_PATH = 'data'
+# Postgres settings
+# DB_TYPE = 'postgres'
+# DB_NAME = 'localhost/SIMPLE'
+# SQLlite settings
+DB_TYPE = 'sqlite'
+DB_NAME = 'SIMPLE.db'
+
+# Set correct connection string
+if DB_TYPE == 'sqlite':
+    # First, remove the existing database in order to recreate it from the schema
+    # If the schema has not changed, this part can be skipped
+    if os.path.exists(DB_NAME):
+        os.remove(DB_NAME)
```

Comment:
Is there a handle for if a user doesn't have write/ remove permissions?

Reply:
Nope, it's meant for you to run on your own machine or by the github action which already has permissions.